src/representation_theory/basic.lean

```diff
@@ -374,6 +374,13 @@ def dual : representation k G (module.dual k V) :=
 @[simp]
 lemma dual_apply (g : G) : (dual ρV) g = module.dual.transpose (ρV g⁻¹) := rfl
 
+/--
+Given `k`-modules `V, W`, there is a homomorphism `φ : V^* ⊗ W → Hom_k(V, W)`
+(implemented by `linear_algebra.contraction.dual_tensor_hom`).
+Given representations of `G` on `V` and `W`,there are representations of `G` on `V^* ⊗ W` and on
+`Hom_k(V, W)`.
```

Comment:
Since this isn't valid lean code, I'd recommend
```suggestion
$Hom_k(V, W)$.
```
Same above for `V^*`.

Since the batch ejected this PR anyway:

bors r-

bors d+